tests/pass-dep/libc/pthread-threadname.rs

```diff
@@ -105,8 +120,12 @@ fn main() {
 
             // But with a too long name it should fail (except on FreeBSD where the
             // function has no return, hence cannot indicate failure).
```

Comment:
```suggestion
            // But with a too long name it should fail (except on FreeBSD where
            // names of arbitrary size seem to be supported).
```
That's what the comment in the shim indicates, at least?